pallets/api/Cargo.toml

```
@@ -0,0 +1,50 @@
+[package]
+name = "pallet-pop-api"
```

```suggestion
name = "pallet-api"
```

Pop prefix adds no value, its obvious based on the repo.